src/pages/workspaces/workspace/Data.js

```diff
@@ -391,10 +391,8 @@ const DataTypeSection = ({ title, titleExtras, error, retryFunction, children })
 ])
 
 const SidebarSeparator = ({ sidebarWidth, setSidebarWidth }) => {
-  const windowWidth = useRef(window.innerWidth)
-
   const minWidth = 280
-  const getMaxWidth = useCallback(() => _.clamp(minWidth, 1200, windowWidth.current - 200), [])
+  const getMaxWidth = useCallback(() => _.clamp(minWidth, 1200, window.innerWidth - 200), [])
```

Comment:
`useCallback` is said to return a memoized version of the callback that only changes if one of the `inputs` has changed. Since there are no inputs in the above case, won't it always return the same function with `window.innerWidth` set to its initial value at all times?

Reply:
That would be the case with something like:
```js
const windowWidth = window.innerWidth
const getMaxWidth = useCallback(() => _.clamp(minWidth, 1200, windowWidth - 200), [])
```

As is, the callback function holds a reference to the `window` object from when the callback function was first created. That's ok though, because the `window` object does not change.

Reply:
Ah interesting.